### PR TITLE
Destructuring

### PR DESCRIPTION
**Pull request by webb04**
Added tip on ES6 destructuring.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,72 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
...
+ // "first"
+ console.log(bar);
+ // "second"
+
+ // you can specify default values
+ const {firstName = "John", lastName: surname = "Doe"} = {};
+  ```
+
+You can find more information about JavaScript ES6 destructuring here:
+* https://strongloop.com/strongblog/getting-started-with-javascript-es6-destructuring/
```
Comment (loverajoel) [inline README.md]: change by [Getting started with javascript es6 destructuring](https://strongloop.com/strongblog/getting-started-with-javascript-es6-destructuring/)

File: README.md [hunk 2]
```diff
@@ -18,6 +18,72 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
...
+* https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/
```
Comment (loverajoel) [inline README.md]: same here

**Comment by loverajoel** [comment]
@webb04 nice tip and so clear

**Comment by Havvy** [comment]
Is basic ES6 functionality really a good idea for tips?

**Comment by loverajoel** [comment]
@webb04 ping!

**Comment by indianakernick** [comment]
This functionality only available in chrome 49 (dev channel). I think this tip is a little early

**Comment by loverajoel** [comment]
Hi, how you can see the jstips repository changed his format and the `master` branch now is `gh-pages`, now are based in _jekyll blog format_. You can see live here => [jstips.com](http://www.jstips.co/)
For that reason you should update your tip with this [instructions](https://github.com/loverajoel/jstips/blob/gh-pages/CONTRIBUTING.md).

Let me know if you need some help or have suggestion. All are welcome!

Thanks

**Comment by zenopopovici** [comment]
@webb04 Please update the PR.

**Comment by webb04** [comment]
@zenopopovici Hi, this has been replaced by #159

**Comment by zenopopovici** [comment]
Thank you @webb04!

**Pull request closed without merging**